### ISSUE TITLE
PhetMenu doesn't work with keyboard

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/joist#543]
Probably related to changes in https://github.com/phetsims/scenery/issues/888, using arrow keys on the PhET menu no longer changes focus, and pressing escape no longer closes the menu.

**Comment by jessegreenberg** [comment]
Just looks like this usage didnt' get the change from DOMEvent -> Scenery Event in the a11y input listener, fixed with the above commit.
